src/components/whole-app/sidebar/nav-page-links/nav-page-links.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExchangeAlt, faFolder, faHome } from '@fortawesome/free-solid-svg-icons';
+import { Nav } from 'react-bootstrap';
+import styles from './nav-page-links.module.css';
+import { useEffect } from 'react';
+
+const NavPageLinks: React.FC = () => {
+  useEffect(() => {
+    const rows = document.getElementsByClassName(styles.row);
+    Array.from(rows).forEach((row) => {
+      const current = row as HTMLElement;
+      if (
+        Array.from(row.children).includes(
+          Array.from(document.getElementsByClassName(styles.active))[0]
+        ) ||
+        current.matches(':hover')
+      ) {
+        current.style.backgroundColor = 'white';
+      } else {
+        current.style.backgroundColor = 'transparent';
+      }
+    });
+  });
+
+  return (
+    <div className={styles.navPageLinks}>
+      <Nav.Item className={styles.row}>
+        <NavLink to="/" exact activeClassName={styles.active} className={styles.container}>
```

Comment:
these `to="..."` parameters should be populated with values from the proper file that defines the routes for our front-end application. This reduces the number of hard-coded string locations and reduces errors.